packages/lib-react-components/src/ZooFooter/components/LogoAndTagline/LogoAndTagline.js

```diff
@@ -0,0 +1,36 @@
+import { Anchor, Box, Image } from 'grommet'
+import { string } from 'prop-types'
+import React from 'react'
+import styled, { ThemeProvider } from 'styled-components'
+
+import ZooniverseLogotype from '../../../ZooniverseLogotype'
+import SpacedText from '../../../SpacedText'
+
+export const StyledLogoAnchor = styled(Anchor)`
+  transition: color 0.2s linear;
+`
+
+export default function LogoAndTagline ({ colorTheme, tagLine }) {
+  const color = colorTheme === 'light' ? 'black' : 'lightGrey'
+  return (
+    <Box>
+      <StyledLogoAnchor
+        color={color}
+        href='https://www.zooniverse.org'
+      >
+        <ZooniverseLogotype id='FooterZooniverseLogo' />
+      </StyledLogoAnchor>
+      <SpacedText
+        color={color}
+        weight='bold'
+      >
```

Comment:
The tagline should be `size='xsmall'` too.

Reply:
Same again - in the classrooms example, the tagline is actually `16px`.